app/preload.js

```diff
@@ -83,56 +83,32 @@ function killLnd() {
   return ipcRenderer.sendSync('killLnd')
 }
 
+/**
+ * Returns specified wallet files location
+ * @param {*} chain
+ * @param {*} network
+ * @param {*} wallet
+ */
+function getWalletDir(chain, network, wallet) {
+  return join(remote.app.getPath('userData'), 'lnd', chain, network, wallet)
+}
+
 /**
  * Delete a local wallet from the filesystem.
  */
-async function deleteLocalWallet(chain, network, wallet, force = false) {
+async function deleteLocalWallet(chain, network, wallet) {
```

Comment:
if we are removing the `force` arg, then it should be removed from the `e2e` test suite where it was being used.

Reply:
done